crypto/pemutil/pem.go

```diff
@@ -418,7 +418,7 @@ func Serialize(in interface{}, opts ...Options) (*pem.Block, error) {
 			}
 		} else {
 			var err error
-			//nolint
+			//nolint:staticcheck // not being used for cryptographic purposes.
```

Comment:
Well, yes, and as we haven't moved to PKCS8 by default, this is the only option.

Reply:
We can convert keys to use PKCS8, but we're not storing them by default.

Reply:
Added another comment.